src/wp-includes/kses.php

```diff
@@ -1256,24 +1256,32 @@ function wp_kses_attr_check( &$name, &$value, &$whole, $vless, $element, $allowe
 	$allowed_attr = $allowed_html[ $element_low ];
 
 	if ( ! isset( $allowed_attr[ $name_low ] ) || '' === $allowed_attr[ $name_low ] ) {
+		$dataset_name = wp_kses_transform_custom_data_attribute_name( $name );
+
+		// Reject custom data attributes that don't fit a basic form.
+		$is_allowable_custom_attribute = (
+			isset( $dataset_name ) &&
```

Comment:
not here because the function must return a string or `null`. `is_string()` would be equally valid.

> Actual question not a passive-aggressive change request.

This is how I assume all questions are when given; thank you though for clarifying. We're working towards the same goal! ✋